Cargo.toml

```diff
@@ -58,6 +58,7 @@ testresult = "0.3.0"
 [features]
 default = []
 postgres = ["warg-server/postgres"]
+not-cli = ["warg-client/not-cli"]
```

Comment:
Negative features are frowned upon; if a crate has multiple (transitive) paths to a dependency (e.g. a direct dependency plus a transitive dependency), the resolved feature set will always be the intersection of all required features, which is likely to cause breakage with negative features.

I'd recommend just making this `cli` (`cli-interactive`?) and adding it to the `default` set.

Reply:
Thanks, will fix

Reply:
Fixed. Changed to `cli-interactive` feature flag, added it to `default` features.

But also added `not-cli-interactive` feature that doesn't include that feature `cli-interactive` since `default` does. Is that the correct way to set that option?

Reply:
When you want to disable any default features you can use `default-features = false` in the dependency spec, sometimes in combination with default features you want to retain, e.g.:

```toml
# crate1
[features]
default = ["foo", "bar"]

# crate2
crate1 = { version = "...", default-features = false, features = ["bar"] } 
```

Reply:
Ah, removing then.

Reply:
Removed